### PR TITLE
🛠 Fix references to jwt secret to include admin version

### LINKED ISSUE #235: Incorrect env variables used for jwt
Depending on the flow of a cbusiness user, they are signed in with a different jwt secret. This was accidentally removed in #224 

If an cbusiness user logs in on the front page they are signed with a normal `jwt_secret`.
If they sign in to their own admin section they are signed with an `admin_jwt_secret`. Login to this area of the app comes with a 5 minute refresh token.

### PR DESCRIPTION
**Pull request by astroash**
- add eslint magic
- update docs to reflect two types of jwt
- include admin_jwt_secret for when cb's login to their admin section

I think I got them all, but let me know if you think anything is missing 😊

Fixes #235 
 #234 #224

### CODE REVIEW

File: docs/config_guide.md [hunk 1]
```diff
@@ -1,7 +1,8 @@
 # Configuration Guide
+
 All configuration is stored in the `config` directory located in the project root.
 
-Secrets and configuration variables that are likely to change between deployment environments are stored in a file called `config.env`. This file should *NOT* be checked into source control.
+Secrets and configuration variables that are likely to change between deployment environments are stored in a file called `config.env`. This file should _NOT_ be checked into source control.
```
Comment (eliascodes) [inline docs/config_guide.md]: Looks like someone hasn't stopped prettier from linting markdown files 🤔 lol

**Comment by eliascodes** [comment]
@astroash are we sticking with admin == cbusiness? 😕

File: docs/config_guide.md (hunk 1 above)
Comment (astroash) [reply docs/config_guide.md]: 🙈 I turned it off half way through this PR

**Comment by astroash** [comment]
> @astroash are we sticking with admin == cbusiness? 😕

@eliascodes there's actually two use cases for the standard `jwt_secret`, it's not just that one is used for cbusinesses and one is used for visitors. I've editted #235 with more details. Let me know if it doesn't make sense.

**Comment by eliascodes** [comment]
@astroash I was talking about the naming convention. You've called the key `admin_jwt_secret`, not `cbusiness_jwt_secret`.

**Comment by astroash** [comment]
@eliascodes yaha, I know that's what you were asking. `jwt_secret` is used on both visitors and cbusiness. cbusinesses have their own admin section to the app, which is when they use a different jwt (`admin_jwt_secret`).

**Comment by eliascodes** [comment]
Lol, but `admin_jwt_secret` will only be used by cbusinesses. If we then have real admin users, what are we going to call their JWT secret?

**Comment by astroash** [comment]
I reckon it's misleading if we call it `cbusiness_jwt_secret`, and have a section on the front page where cbusinesses login with `jwt_secret` not the `cbusiness_jwt_secret`. Maybe they could be called `standard_jwt_secret`, `cbusiness_admin_jwt_secret` & `twine_admin_secret` (for when we add the third flow)?

**Comment by eliascodes** [comment]
> `standard_jwt_secret`, `cbusiness_admin_jwt_secret` & `twine_admin_secret`

That's closer. See [my comment](https://github.com/TwinePlatform/DataPower/issues/234#issuecomment-369298163)

**Pull request merged**